### PR TITLE
Soil temp

### PR DESCRIPTION
**Pull request by robertson-mark**
Snotel ground temps should be fairly set. Mesowest and CDEC metadata are a bit lacking, so we may need to update the variables depending on that. No tests written.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -70,18 +70,25 @@ get started contributing to metloom!
...
 
 
-    MyCDECPointData("TNY", "Tenaya Lake").get_daily_data(
-        datetime(2020, 1, 3), datetime(2020, 1, 7), [MyVariables.DEWPT]
-    )
+    variables = [MyVariables.RH, MyVariables.WINDSP]
+    stn = MyCDECPointData("TNY", "Tenaya Lake")
+    df = stn.get_daily_data(datetime(2021, 12, 25), datetime(2021, 12, 26), variables)
+
+    print(df[['Relative Humidity', 'Wind Speed']])
```
Comment (micah-prime) [inline docs/usage.rst:94]: This is a nice update to the docs!

File: metloom/pointdata/snotel.py [hunk 2]
```diff
@@ -44,7 +43,7 @@ def __init__(self, station_id, name, metadata=None):
         self._tzinfo = None
 
     def _snotel_response_to_df(self, result_map: Dict[SensorDescription, List[dict]],
-                               duration: str, include_measurement_date=False):
+                               duration: str, include_measurement_date=False, **kwargs):
```
Comment (micah-prime) [inline metloom/pointdata/snotel.py:46]: This **kwargs doesn't look like it is getting used

File: metloom/variables.py [hunk 3]
```diff
@@ -73,6 +73,21 @@ class CdecStationVariables(VariableBase):
...
+    )
+    TEMPGROUND4 = SensorDescription(
+        "195", "GROUND TEMPERATURE -4", "GROUND TEMPERATURE OBS -4in"
+    )
+    TEMPGROUND8 = SensorDescription(
+        "196", "GROUND TEMPERATURE -8", "GROUND TEMPERATURE OBS -8in"
+    )
+    SOLARRAD = SensorDescription("103", "Solar Radiation", "SOLAR RADIATION")
+    WINDSPEED = SensorDescription("9", "Wind Speed", "WIND SPEED")
+    WINDDIR = SensorDescription("10", "Wind Direction", "WIND DIRECTION")
```
Comment (micah-prime) [inline metloom/variables.py:90]: Can you change the SensorDescription.name (the second argument) to be all caps for these entries

File: metloom/pointdata/snotel.py [hunk 4]
```diff
@@ -226,6 +239,32 @@ def _get_tzinfo(self):
...
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND4.name:
+                extra_params.update(
+                    {variable.name: {'height_depth': {"value": -4, "unitCd": "in"}}}
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND8.name:
+                extra_params.update(
+                    {variable.name: {'height_depth': {"value": -8, "unitCd": "in"}}}
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND20.name:
```
Comment (micah-prime) [inline metloom/pointdata/snotel.py:262]: You should be able to drop `.name` from these comparisons (i.e. `if variable == self.ALLOWED_VARIABLES.TEMPGROUND20`)

If you want to simplify further I think the function could look like this
```

        extra_params_map = {
            self.ALLOWED_VARIABLES.TEMPGROUND2: {
                'height_depth': {"value": -2, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND4: {
                'height_depth': {"value": -4, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND8: {
                'height_depth': {"value": -8, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND20: {
                'height_depth': {"value": -20, "unitCd": "in"}
            },

        }

        extra_params = {}
        for variable in variables:
            result = extra_params_map.get(variable)
            if result:
                extra_params[variable.name] = result

        return extra_params or None
```

**Review by micah-prime — CHANGES_REQUESTED** [review]
This looks really good! Just a couple small changes and then we should be good to go

File: metloom/pointdata/snotel.py (hunk 2 above)
Comment (robertson-mark) [reply metloom/pointdata/snotel.py:46]: thanks, I also needed to remove passing the kwargs to this function, which was being done but not needed

File: metloom/pointdata/snotel.py (hunk 4 above)
Comment (robertson-mark) [reply metloom/pointdata/snotel.py]: done!

File: metloom/pointdata/snotel.py (hunk 2 above)
Comment (micah-prime) [reply metloom/pointdata/snotel.py:46]: @robertson-mark I think this is the last outstanding thing then we should be good to merge

Comment (robertson-mark) [reply metloom/pointdata/snotel.py]: thanks, should be good now

**Review by micah-prime — APPROVED** [review]

**Pull request merged**